nixos/modules/services/web-apps/immich.nix

```diff
@@ -116,7 +116,7 @@ in
       description = ''
         Configuration for Immich.
         See <https://immich.app/docs/install/config-file/> or navigate to
-        <https://your-immich-domain/admin/system-settings> for
+        <https://my.immich.app/admin/system-settings> for
```

Comment:
why this change? should we be linking to their site? I think the placeholder is fine.

Reply:
The whole purpose of https://my.immich.app/ is to redirect to your Immich instance.

Reply:
ok :shrug: